README.md

```diff
@@ -1,6 +1,8 @@
 # okta-oidc-js
 This is a monorepo that contains Okta's OpenID Connect JavaScript resources.
 
+Contributors should read our [contributing guidelines](./CONTRIBUTING.md) if they wish to contribute.
```

Comment:
This section can be removed, as it is already at the footer in our README: <https://github.com/okta/okta-oidc-js#contributing>